changelog/unreleased/bugfixes/6540.md

```diff
@@ -0,0 +1 @@
+Fixed approaches list update in `RouteOptionsUpdater`(uses for reroute). It was putting to the origin approach corresponding approach from legacy approach list.
```

Comment:
Shouldn't there be a `-` in the beginning?

Reply:
I think the script will add `-` during assembling

Reply:
It's not a very good idea. If the entry is multi-line, how will the script understand where to add the `-`?

Reply:
if you have only one feture/bugfix you can don't use `-` before, if you have more you should